src/frontend/sharedComponents/BottomSheetModal/Backdrop.tsx

```diff
@@ -13,7 +13,7 @@ export const Backdrop = ({
     () =>
       interpolate(animatedIndex, {
         inputRange: [0, 1],
-        outputRange: [0, 0.3],
+        outputRange: [0.1, 0.3],
```

Comment:
i don't think we want to change this. having this would cause the backdrop to always be visible at 0.1 opacity, even if the sheet is closed

Reply:
right, i was just messing around, ill put this back